com.jpexs.decompiler.flash.json

```diff
@@ -0,0 +1,58 @@
+{
+	"app-id": "com.jpexs.decompiler.flash",
+	"runtime": "org.freedesktop.Platform",
+	"runtime-version": "20.08",
+	"sdk": "org.freedesktop.Sdk",
+	"sdk-extensions": [
+		"org.freedesktop.Sdk.Extension.openjdk8"
+	],
+	"command": "ffdec.sh",
+	"finish-args": [
+		"--socket=x11",
+		"--env=PATH=/app/bin:/app/jre/bin:/usr/bin",
+		"--env=JAVA_HOME=/app/jre",
+		"--filesystem=host"
+	],
+	"modules": [
+		{
+			"name": "openjdk",
+			"buildsystem": "simple",
+			"build-commands": [
+				"/usr/lib/sdk/openjdk8/install.sh"
+			]
+		},
+		{
+			"name": "jpexs.decompiler.flash",
+			"buildsystem": "simple",
+			"build-commands": [
+				"ar x ffdec_14.4.0.deb",
+				"rm -f ffdec_14.4.0.deb",
+				"tar -xf data.tar.gz",
+				"rm -f control.tar.gz data.tar.gz debian-binary",
+				"rm usr/share/java/ffdec/ffdec.{bat,exe}",
+				"install -Dm644 usr/share/applications/FFDec.desktop /app/share/applications/com.jpexs.decompiler.flash.desktop",
+				"install -Dm644 usr/share/java/ffdec/icon.png /app/share/icons/hicolor/512x512/apps/com.jpexs.decompiler.flash.png",
+				"install -Dm644 com.jpexs.decompiler.flash.appdata.xml /app/share/appdata/com.jpexs.decompiler.flash.appdata.xml",
+				"mkdir -p /app/bin",
+				"mv usr/share/java/ffdec/* /app/bin",
+				"desktop-file-edit --set-key=Exec --set-value=ffdec.sh /app/share/applications/com.jpexs.decompiler.flash.desktop",
+				"desktop-file-edit --set-key=Icon --set-value=com.jpexs.decompiler.flash /app/share/applications/com.jpexs.decompiler.flash.desktop",
+				"chmod +x /app/bin/ffdec.sh"
+			],
+			"sources": [
+				{
+					"type": "file",
+					"only-arches": [
+						"x86_64"
+					],
+					"url": "https://github.com/jindrapetrik/jpexs-decompiler/releases/download/version14.4.0/ffdec_14.4.0.deb",
+					"sha256": "72fc1582d42a49ac175f6a317f3f804e2c3662ec3f0ad1ec2cbfb754c4472daa"
```

Comment:
it's GPL software, why not rebuilding from source and add aarch64 support? Does this package even contain non Java binaries since there seem to be a java only zip.

Reply:
Oh, you're thinking it should be usable on aarch64 too since there's no native code? That is very possible.

Reply:
Let's do it. At first glance it looks like it should work just fine, although I haven't checked on ARM hardware.

Reply:
> Let's do it

Could you clarify what you're proposing I should do?

Reply:
I think removing `flathub.json` will be enough.